tests/Integration/LongRunning/long_running_script_manual.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use DDTrace\GlobalTracer;
+use DDTrace\Tag;
+
+function do_manual_instrumentation()
+{
+    $tracer = GlobalTracer::get();
+    $rootSpan = $tracer->startActiveSpan("root-operation")->getSpan();
+    $rootSpan->setTag(Tag::SERVICE_NAME, "long-running-service");
+    $rootSpan->setTag(Tag::RESOURCE_NAME, "root-resource");
+
+    $subSpan = $tracer->startActiveSpan('sub-operation')->getSpan();
+    $rootSpan->setTag(Tag::RESOURCE_NAME, "sub-resource");
+    $subSpan->finish();
+    $rootSpan->finish();
+
+    if (PHP_MAJOR_VERSION === 5) {
```

Comment:
@SammyK is this OK or we should not recommend (and mention in docs) tracing of long running scripts on PHP 5?

Reply:
As long as the userland API is being used (as in this example), this should be fine. We just need to add `$tracer->reset();` after the flush below.

Reply:
Updated our card. I am skipping the test on PHP 5 and we are not going to suggest long running scripts on 5 until we properly and extensively test it either manually or on our reliability environment.